packages/api/controllers/hackathon_register_controller.go

```diff
@@ -152,3 +161,49 @@ func FormatTeamName(teamName string) string {
 func CompareTeamNames(passedTeamName string, storedTeamName string) bool {
 	return FormatTeamName(passedTeamName) == FormatTeamName(storedTeamName)
 }
+
+func SendEmailToNewParticipant(fullName string, email string) {
+
+	requestURL := os.Getenv("MAILING_URL")
+
+	type Mailer struct {
+		Receiver string `json:"receiver"`
+		Html     string `json:"html"`
+		Subject  string `json:"subject"`
+	}
+
+	var reqBody Mailer
+
+	reqBody.Html = "<h1>HELLO</h1>"
```

Comment:
This should be a dynamically loaded template. I'm attaching last year's template here: https://github.com/AUBGTheHUB/website-2021/blob/3e5acfc89cd27b529da295748805838af42b1314/packages/api/src/templates/confirmation.html

Instead of "Dear Participant", make it "Dear {name}"